### DISCUSSION TITLE
Is an Eufy paid for subscription required?

### DISCUSSION THREAD

**Discussion by st2000** [fuatakgun/eufy_security#1200] [General]
I love the "How Does This Work" section of the README!

It does not state the annual Eufy subscription is necessary.

About half the Eufy owners I know do not pay for an annual subscription. This is one of the positive reasons to buy an Eufy. Since a 2nd login is required to make this application work, I assume the logins are for the paid subscription? So, no subscription means this application will not be able to access Eufy video?

**Comment by fuatakgun** [comment]
no, paid subscription is not required for this add-on, you can share your devices with any account and creating a secondary account and sharing your devices with new account would be enough.

**Reply by piwi447** [reply]
Hello, I am searching the way to create this second eufy-security account and then share my devices to this account. Could you please explain how you proceed to do this. I have Eufy security app installed on two iPhones and two iPads but I cannot find any menu in the app that allows this? On the Eufy security website, they only explain how to create a new account from the app when you connect the first time . What did I misses? I installed my eufy cam 2 years ago and I don’t remember every details of the account setup…
Thanks for any help
